src/pip/_internal/commands/cache.py

```diff
@@ -96,18 +96,19 @@ def get_cache_info(self, options: Values, args: List[Any]) -> None:
         http_cache_location = self._cache_dir(options, "http-v2")
         old_http_cache_location = self._cache_dir(options, "http")
         wheels_cache_location = self._cache_dir(options, "wheels")
-        http_cache_size = filesystem.format_directory_size(http_cache_location)
-        old_http_cache_size = filesystem.format_directory_size(old_http_cache_location)
+        http_cache_size = (
+            filesystem.format_directory_size(http_cache_location) +
+            filesystem.format_directory_size(old_http_cache_location)
```

Comment:
```suggestion
            filesystem.format_size(filesystem.directory_size(http_cache_location) +
            filesystem.directory_size(old_http_cache_location))
```

With the relevant rewrapping, to avoid output like "2kB2kB".

Reply:
Oops, thanks for catching that.